packages/cli/src/commands/create-test/create-test.command.ts

```diff
@@ -204,10 +217,7 @@ export const createTest: CommandModule<unknown, Options> = {
       description:
         "Query selector to screenshot a specific DOM element instead of the whole page",
     },
-    moveBeforeClick: {
-      boolean: true,
-      description: "Simulate mouse movement before clicking",
-    },
+    moveBeforeClick: OPTIONS.moveBeforeClick,
```

Comment:
Fixing the types caught a bug that we weren't providing the default value for moveBeforeClick.